warehouse/models/mart/gtfs_quality/_mart_gtfs_quality.yml

```diff
@@ -549,6 +549,32 @@ models:
           Boolean for whether this file was present for all feeds for this organization
           on `date_checked`.
 
+  - name: fct_monthly_route_id_changes
+    description: |
+      A table that quantifies how many routes in an organization's schedule were added, removed, or unchanged
+      over the course of a month by comparing the routes scheduled on the first day of the month
```

Comment:
nitpick here: it's technically not `routes scheduled on the first day of the month` but `routes present in the feed that was active on the first of the month`.... def a subtle difference but could make a difference ex. if the first of the month is a weekend the scheduled routes will be fewer but the routes present will stay the same